prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.html

```diff
@@ -16,11 +16,32 @@
 
   <div class="row mt-4">
     <div *ngIf="enrolment?.enrollee.gpid"
-         class="col-12 pb-3 text-blue-light">
+         class="col-12 d-flex justify-content-between align-items-center">
```

Comment:
Fixed, the CSS class was not used after our discussion I just forgot to remove it from the file. One note though, the Admin Actions and TOA dropdowns were sticking out to the right by 15pixels from the start So no I am not sure if that was an error or if it was supposed to be like that and if it should still stick out of the row. For now it's within the same row as the GPID and it's not sticking out.

Reply:
Changed. with the `mt-4` class in the containing div (`div class="d-flex justify-content-between align-items-center mt-4"`) , there's a lot of white space noticeable between the table and the GPID/TOA/Adjudicator Actions, and so there is big inconsistent gap between the table and the row and the Enrolment and Adjudication cards which is not consistent. We can add an `mt-4` class to each of the cards to make them look consistent, what do you think?

Reply:
Good rule is to never add a margin or padding to the top of an element that iterate or you intend to stack within a container always add it to the bottom, which is easier to reuse and maintain.  The margin-top is added to this element since it will always be the first child in the container.  Also, see my other comment you're still mixing grid and flexbox utilities together.

Reply:
Fixed. Was the <br> between `GPID:` label and its value intended or an oversight?

Reply:
Oversight.  I've removed it.